Yoast/wordpress-seo
# Horizontal metabox menu

## Summary

<img width="518" alt="Screenshot 2019-06-05 at 11 26 08" src="https://user-images.githubusercontent.com/5389513/58945753-c27e4a00-8784-11e9-894c-5d2129f548e5.png">

Todo:
* [x] Remove tooltips
* [x] Remove screen reader labels in favour of visible labels
* [x] Add labels in taxonomy metabox tabs

This PR can be summarized in the following changelog entry:

* Changes the tabs in the metabox to horizontal tabs.

## Relevant technical choices:

* Responsive view will be fixed in a separate PR.
* Creating an SEO and readability tab will occur in a future PR: https://github.com/Yoast/wordpress-seo/issues/12916, as well as chaging the icons.

## Test instructions

This PR can be tested by following these steps:

* Make sure the tabs are according to the designs: https://github.com/Yoast/design/issues/383
* Check the posts/pages metabox, make sure the horizontal tabs work properly. No need to check for how they function in mobile view. This will be solved in a separate PR.
* Check the taxonomies metabox.
* Also test with one or multiple add-ons.

## UI changes

## Quality assurance


Fixes #12915
